content/en/docs/tasks/tls/certificate-rotation.md

```diff
@@ -10,12 +10,12 @@ content_template: templates/task
 This page shows how to enable and configure certificate rotation for the kubelet.
 {{% /capture %}}
 
+{{< feature-state for_k8s_version="v1.8" state="beta" >}}
+
 {{% capture prerequisites %}}
 
 * Kubernetes version 1.8.0 or later is required
```

Comment:
Since v1.8 is out of support, it's OK to adjust the page to drop mentioning v1.8.0 as a prerequisite. You can instead
use:
```markdown
{{< include "task-tutorial-prereqs.md" >}}
```

Reply:
(the `feature-state` shortcode should stay)

Reply:
(the `feature-state` shortcode should stay; that's a useful piece of information to show that the feature is beta and has been for a while)